modules/cudastereo/src/cuda/stereosgm.hpp

```diff
@@ -0,0 +1,61 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Author: The "SGM Team" at Fixstars Corporation
+
+#ifndef OPENCV_CUDASTEREO_SGM_HPP
+#define OPENCV_CUDASTEREO_SGM_HPP
+
+#include "opencv2/core/cuda.hpp"
+
+namespace cv { namespace cuda { namespace device {
+namespace stereosgm
+{
+
+namespace census_transform
+{
+CV_EXPORTS void censusTransform(const GpuMat& src, GpuMat& dest, cv::cuda::Stream& stream);
```

Comment:
Do we need `CV_EXPORTS` ?

Reply:
In Windows environment, `CV_EXPORTS` would be necessary to build `test_sgm_funcs.cpp` of `opencv_test_cudastereo` project if `BUILD_SHARED_LIBS` was set to `ON`.
I also feel that it is not proper way...

Reply:
OK, I see.